packages/pwa/app/pages/home/index.jsx

```diff
@@ -37,90 +46,204 @@ const Home = () => {
+                mx="auto"
+                py={{base: 8, md: 16}}
+                px={{base: 4, md: 8}}
+                borderRadius="base"
+                width={{base: '100vw', md: 'inherit'}}
+                position={{base: 'relative', md: 'inherit'}}
+                left={{base: '50%', md: 'inherit'}}
+                right={{base: '50%', md: 'inherit'}}
+                marginLeft={{base: '-50vw', md: 'auto'}}
+                marginRight={{base: '-50vw', md: 'auto'}}
             >
-                <Grid
-                    templateRows={{base: 'repeat(1, 1fr)', md: 'repeat(auto, 1fr)'}}
-                    templateColumns={{base: 'repeat(1, 1fr)', md: 'repeat(6, 1fr)'}}
-                    columnGap={6}
-                    rowGap={8}
+                <SimpleGrid
+                    columns={{base: 1, md: 1, lg: 3}}
+                    spacingX={{base: 1, md: 4}}
+                    spacingY={{base: 4, md: 14}}
                 >
-                    {categoriesThreeColumns.map((item, index) => {
-                        const category = item.message
+                    {heroFeatures.map((feature, index) => {
+                        const featureMessage = feature.message
                         return (
-                            <GridItem key={index} rowSpan={1} colSpan={{base: 1, md: 2}}>
-                                <BasicTile
-                                    title={intl.formatMessage(category.title)}
-                                    href={intl.formatMessage(category.href, {
-                                        activeLocale: intl.locale
-                                    })}
-                                    img={{
-                                        src: getAssetUrl(intl.formatMessage(category.imgSrc)),
-                                        alt: intl.formatMessage(category.imgAlt)
-                                    }}
-                                />
-                            </GridItem>
+                            <Box
+                                key={index}
+                                bg={'white'}
+                                boxShadow={'0px 2px 2px rgba(0, 0, 0, 0.1)'}
+                                borderRadius={'4px'}
+                            >
+                                <Link target="_blank" href={feature.href}>
+                                    <HStack>
+                                        <Flex pl={6} h={24} align={'center'} justify={'center'}>
+                                            {feature.icon}
+                                        </Flex>
+                                        <VStack align={'start'}>
+                                            <Text fontWeight="700">
+                                                {intl.formatMessage(featureMessage.title)}
+                                            </Text>
+                                        </VStack>
+                                    </HStack>
```

Comment:
Minor: do we really need the VStack? It looks like we can safely remove it.

```suggestion
                                    <HStack>
                                        <Flex pl={6} h={24} align={'center'} justify={'center'}>
                                            {feature.icon}
                                        </Flex>
                                        <Text fontWeight="700">
                                            {intl.formatMessage(featureMessage.title)}
                                        </Text>
                                    </HStack>
```